src/domain/safe/safe.repository.ts

```diff
@@ -449,6 +449,45 @@ export class SafeRepository implements ISafeRepository {
     }, {});
   }
 
+  async getAllSafesByOwner(args: {
+    ownerAddress: `0x${string}`;
+  }): Promise<{ [chainId: string]: Array<string> | null }> {
+    const chains = await this.chainsRepository.getAllChains();
+    const allSafeLists = await Promise.allSettled(
+      chains.map(async ({ chainId }) => {
+        const safeList = await this.getSafesByOwner({
+          chainId,
+          ownerAddress: args.ownerAddress,
+        });
+
+        return {
+          chainId,
+          safeList,
+        };
+      }),
+    );
+
+    const result: { [chainId: string]: Array<string> | null } = {};
+
+    for (const [index, allSafeList] of allSafeLists.entries()) {
+      const chainId = chains[index].chainId;
+
+      if (allSafeList.status === 'rejected') {
+        this.loggingService.warn(
+          `Failed to fetch Safe owners. chainId=${chainId}`,
+        );
+      }
+
+      result[chainId] =
+        allSafeList.status === 'fulfilled'
+          ? allSafeList.value.safeList.safes
+          : // Transaction Service threw; could own Safes or not
+            null;
+    }
```

Comment:
The same check twice.
```suggestion
      if (allSafeList.status === 'fulfilled') {
        result[chainId] = allSafeList.value.safeList.safes
      } else {
        result[chainId] = null
        this.loggingService.warn(
          `Failed to fetch Safe owners. chainId=${chainId}`,
        );
      }
    }
```

Reply:
Changed in 2e27da02514d37577621737f2a9d10968172e559.